src/uct/api/uct.h

```diff
@@ -261,9 +264,10 @@ typedef enum {
  * @brief  Asynchronous event types.
  */
 enum uct_iface_event_types {
-    UCT_EVENT_SEND_COMP   = UCS_BIT(0), /**< Send completion event */
-    UCT_EVENT_RECV_AM     = UCS_BIT(1), /**< Active message received */
-    UCT_EVENT_RECV_SIG_AM = UCS_BIT(2)  /**< Signaled active message received */
+    UCT_EVENT_SEND_COMP     = UCS_BIT(0), /**< Send completion event */
+    UCT_EVENT_RECV_COMP     = UCS_BIT(1), /**< Tag or active message received */
```

Comment:
maybe just UCT_EVENT_RECV? it's not the same as the send side, when send can be started and completed later.